mimic/model/maas_objects.py

```diff
@@ -0,0 +1,229 @@
+"""
+MaaS API data model
+"""
+
+import random
+
+
+def random_letters(len_min, len_max):
```

Comment:
Any reason we cant use https://github.com/rackerlabs/mimic/blob/master/mimic/util/helper.py#l37